teamscale-gradle-plugin/build.gradle

```diff
@@ -3,7 +3,6 @@ plugins {
 	id 'java'
 	id 'java-gradle-plugin'
 	id 'maven'
-	id 'com.github.johnrengelman.shadow' version '4.0.3'
```

Comment:
Not sure if we can get rid of the shadowing for the plugin. We might (as we already did in the past) run into version conflicts with e.g. the jgit library used inside Gradle.

Reply:
But for me it would also be ok for now to live with the risk and fix it whenever a Gradle update breaks it.

Reply:
My gut tells me that we should try to only include any lines of code that are strictly required for our project to work. Hence I'd argue to keep it this way for now and see if we run into any problems we should explicitly handle. However I'm not quite sure if this is right approach. Do you think we can run into bugs where coverage is incorrect or our plugin badly interferes with the dependencies of other Gradle plugins?